l10n_it_website_portal_fatturapa_sale/controllers/main.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright 2019 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import http
+from odoo.exceptions import ValidationError
+from odoo.http import request
+from odoo.addons.sale.controllers.portal import CustomerPortal
+
+
+class CustomerPortalFatturapaSale(CustomerPortal):
+
+    @http.route(['/my/orders/<int:order_id>/transaction/'],
+                type='json', auth="public", website=True)
+    def payment_transaction_token(
+            self, acquirer_id, order_id, save_token=False, access_token=None,
+            **kwargs):
+
+        partner = request.env.user.partner_id
+        try:
+            partner.update({
+                'electronic_invoice_subjected': True,
+                'customer': True,
+            })
```

Comment:
Se non capisco male, in questo modo, installando questo modulo, che è `auto_install`, tutti i clienti verrebbero bloccati, anche i non italiani.

Inoltre, essendo il modulo pensato, mi pare di capire, per uno scenario tipicamente B2B (altrimenti c'è `l10n_it_website_sale_fatturapa`), dove quindi il cliente in odoo viene creato da un utente di backend, l'utente di backend dovrebbe aver già eventualmente impostato `electronic_invoice_subjected` e quindi i dati per la fatturazione elettronica.

Quindi qual è lo scenario?

Reply:
Ciao, grazie della review! 
Ho aggiunto un check nel popup del pagamento per chiedere all'utente se vuole fattura elettronica, puoi vedere se ora ti convince di più?

Lo scenario è che l'utente di backend ha creato il partner senza sapere se è soggetto a fattura elettronica.